cmd/mistry/main.go

```diff
@@ -248,30 +249,37 @@ EXAMPLES:
 					return err
 				}
 
+				if !jsonResult {
+					fmt.Println("Logs can be found at", baseUrl+"/"+bi.URL)
+				}
+
 				if verbose {
 					fmt.Printf(
 						"\nResult:\nStarted at: %s ExitCode: %v Params: %s Cached: %v Coalesced: %v\n\nLogs:\n%s\n",
 						bi.StartedAt, bi.ExitCode, bi.Params, bi.Cached, bi.Coalesced, bi.Log)
 				}
 
 				if jsonResult {
-					fmt.Printf("%s", body)
+					fmt.Printf("%s\n", body)
 				}
 
 				if bi.ExitCode != 0 {
+					if bi.ErrLog != "" {
+						fmt.Fprintln(os.Stderr, "Container error logs:\n", bi.ErrLog)
```

Comment:
Maybe we should print this line even if `bi.ErrLog` is empty, so that it's apparent that there were actually no error logs?

As it is now, one might wonder why container error logs were not displayed.